config/routes.rb

```diff
@@ -56,6 +56,7 @@
   get  'followees', to: 'issues#followees'
   get  'feed', to: 'feeds#common_feed'
   get  'user_feed', to: 'feeds#user_feed'
+  get  'statistics', to: 'common_pages#statistics'
```

Comment:
maybe add some spec to controller, just to make sure it's more stable, with some params which you have added